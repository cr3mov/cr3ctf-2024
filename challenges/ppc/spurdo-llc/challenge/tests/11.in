100 42357203260
1210131637 1801964491 1328730755 369781743 20000000 1955945601 1819274209 763133449 17589072 20000000 600849409 1896225113 20000000 1807335 973691843 20000000 20000000 558665359 20000000 1644996892 2181926 20000000 20000000 20000000 807929680 20000000 20000000 686913368 20000000 20000000 20000000 14940825 20000000 20000000 20000000 20000000 1119769899 20000000 17838527 20000000 20000000 1238368449 1898745751 1483674423 18764937 20000000 8901114 1844977559 878468712 20000000 20000000 20000000 735383447 35582108 1409530982 369665446 20000000 20000000 729000110 20000000 20000000 430591002 5527483 20000000 59194531 20000000 3649366 10426762 1574419439 20000000 20000000 564836073 365104262 1872750825 20000000 11709167 1456069366 1185213139 20000000 238819484 3887998 12459159 483676572 20000000 392740207 20000000 1732535180 20000000 1298623877 20000000 798973421 20000000 1146037808 20000000 20000000 20000000 20000000 20000000 1416217595 6666476
