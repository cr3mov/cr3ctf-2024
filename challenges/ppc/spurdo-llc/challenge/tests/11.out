19999999 19999999 19999999 19999999 19999999 19999999 19999999 19999999 17589072 19999999 19999999 19999999 19999999 1807335 19999999 19999999 19999999 19999999 19999999 19999999 2181926 19999999 19999999 19999999 19999999 19999999 19999999 19999999 19999999 19999999 19999999 14940825 19999999 19999999 19999999 19999999 19999999 19999999 17838527 19999999 19999999 19999999 19999999 19999999 18764937 19999999 8901114 19999999 19999999 19999999 19999999 20000000 20000000 20000000 20000000 20000000 20000000 20000000 20000000 20000000 20000000 20000000 5527483 20000000 20000000 20000000 3649366 10426762 20000000 20000000 20000000 20000000 20000000 20000000 20000000 11709167 20000000 20000000 20000000 20000000 3887998 12459159 20000000 20000000 20000000 20000000 20000000 20000000 20000000 20000000 20000000 20000000 20000000 20000000 20000000 20000000 20000000 20000000 20000000 6666476 